### ISSUE TITLE
The fluctuation of evaluation indicators

### ISSUE THREAD

**Issue by zhegemimahaolihai666** [nilesh2797/DEXML#8]
hi, @nilesh2797 ,
When I trained using the English dataset, everything went smoothly.
However, when training with the Chinese Extreme Multi-label Dataset (with 100,000 pieces of data and 200 labels), I found that the P@1 metric was 68% for the first time, and when epoch = 5, the P@1 metric was 10% for the first time.
The tf parameter used is bert-base-chinese, and the specific picture of the training with the Chinese dataset is as follows:

<img width="1458" alt="Image" src="https://github.com/user-attachments/assets/f16b15df-f621-46fa-8851-d7c6adbd2458" />

Do you know the specific reason for this?
To conform to the training method of the model, I aligned the formats of the Chinese dataset and the English dataset.
But there are still problems with the Chinese training.
Do you know the reason? Why does the evaluation result fluctuate so much?
Does the Chinese dataset require special processing?

**Comment by zhegemimahaolihai666** [comment]
The following is my configuration file.

# Startup command
accelerate launch --config_file configs/accelerate.yaml --num_processes 8 train.py ./dist-asym-zslwan_softmax.yaml --dataset title_tags --tau 0.5 --dropout 0.2 --tf_max_len 512 --loss_criterion softmax --warmup 0.2 --weight_decay 0.01

# dist-asym-zslwan_softmax.yaml
__dependency__: configs/dual-encoder.yaml, configs/EURLEX57K/dataset.yaml
expname: dist-asym-zslwan-distilbert_sfr_softmax_ep50_bs32_lr5e-5_exp
desc: "Training dist-asym-zslwan (distributed asymmetric zero-shot label-wise attention) with softmax loss"


net: dist-asym-zslwan
tf: /data/multi_tags_model/new_tags_de/ZSLWAN-release/bert-base-chinese
tf_pooler: seq
lbl_embs: Datasets/title_tags/lbl_embs_gte.npy
norm_embs: False
tau: 0.05
neg_type: none # dist-de-all nets uses all labels as negatives, so no need to sample negatives in data loading
loss_criterion: softmax
loss_sample: False


num_epochs: 100
bsz: 32
gc_bsz: 4096
eval_interval: 5
lr: 1.e-5
clf_lr: 1.e-4
dropout: 0.1

# Eval Result

<img width="1612" alt="Image" src="https://github.com/user-attachments/assets/224636e4-7a3c-464a-877d-770f53812249" />
